modules/ROOT/pages/mongodb/mongodb-connector-6-0-examples.adoc

```diff
@@ -0,0 +1,112 @@
+= MongoDB Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+== Invoke an Operation
+
+To invoke a simple operation, such as the *Insert Document* operation, follow these steps:
+
+. In Studio, click *HTTP* and drag the *Listener* operation to the canvas.
+. From the palette, select *MongoDB* and drag the *Insert Document* operation into your flow by placing it to the right of HTTP Listener.
+. Configure the connector by selecting the *Connector Configuration* you created in the previous section and choosing the operation to invoke and set the *Collection name*.
+. From the palette, drag the *Transform Message* component before the MongoDB connector.
+. Click on *Transform Message* and enter two key-value pairs:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+%output application/json
+---
+{
+	name:"Peter",
+	age:"42"
+}
+----
++
+. From the Mule Palette view, drag other *Transform Message* components after the MongoDB connector.
+. Click *Transform Message* and set the output to `application/json`.
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload
+----
++
+. Other possible flow options are:
+*Reviewers, what is this trying to say? There are many other operations, so why is this here?*
```

Comment:
Yes, this is confusing. The idea was to put operations related to the Documents, but perhaps it is better to delete this point.